sagemaker/launch/launch_graphbolt_convert.py

```diff
@@ -0,0 +1,124 @@
+""" Launch SageMaker training task
+"""
+import logging
+import os
+from time import strftime, gmtime
+
+import boto3 # pylint: disable=import-error
+import sagemaker
+from sagemaker.processing import ScriptProcessor
+from sagemaker.workflow.steps import ProcessingInput
+
+from common_parser import ( # pylint: disable=wrong-import-order
+    get_common_parser, parse_estimator_kwargs)
+
+INSTANCE_TYPE = "ml.m5.12xlarge"
+
+SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
+
+def run_gbconvert_job(input_args, image):
+    """ Run job using SageMaker estimator.PyTorch
+
+        TODO: We may need to simplify the argument list. We can use a config object.
+
+    Parameters
+    ----------
+    input_args:
+        Input arguments
+    image: str
+        ECR image uri
+    """
+    timestamp = strftime("%Y-%m-%d-%H-%M-%S", gmtime())
+    # SageMaker base job name
+    sm_task_name = input_args.task_name if input_args.task_name else timestamp
```

Comment:
Actually we do this with base_job_name on line 55. I'll make a change so we use the user-provided string if provided, otherwise just have `gbconvert` as base name, and then SM automatically adds a timestap to ensure uniqueness